deploy/compose/compose-testing.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  chromedriver:
+    image: $CHROMEDRIVER_IMAGE
+    ports:
+      - "4444:4444"
+    shm_size: "2G"
+    network_mode: host
```

Comment:
Can we name this file, `compose-ui-testing.yaml` or something else that distinguishes it from the units/integs?